lib/client.js

```diff
@@ -28,7 +28,7 @@ class Client extends Entity {
     super(handle, typeClass, options);
     this._nodeHandle = nodeHandle;
     this._serviceName = serviceName;
-    this._sequenceNumber = 0;
+    this._pending_requests = new Map();
```

Comment:
I would suggest renaming it to `this._senquenceNumberToCallbackMap` to align with the JS naming convention

Reply:
Thanks for pointing this out ! I just pushed a commit with the change